pymorphy2/units/base.py

```diff
@@ -10,6 +9,21 @@
 )
 
 
+try:
```

Comment:
Why don't you use `sys.version_info < (3,)` here? There is no point in using the deprecated function in Python 3. The speed is not a concern here either.

Reply:
Yep, could also check version_info. I guess decided to apply same pattern as in other files
https://github.com/pymorphy2/pymorphy2/blob/master/pymorphy2/dawg.py#L4
https://github.com/pymorphy2/pymorphy2/blob/master/pymorphy2/tagset.py#L8
https://github.com/pymorphy2/pymorphy2/blob/master/pymorphy2/opencorpora_dict/parse.py#L19

Reply:
Fair point. Nonetheless, I don't consider that to be the best solution here, just because the function is long deprecated. Nothing more than a matter of taste.